src/Runner.Common/Constants.cs

```diff
@@ -227,6 +227,7 @@ public static class Actions
                 //
                 public static readonly string AllowUnsupportedCommands = "ACTIONS_ALLOW_UNSECURE_COMMANDS";
                 public static readonly string AllowUnsupportedStopCommandTokens = "ACTIONS_ALLOW_UNSECURE_STOPCOMMAND_TOKENS";
+                public static readonly string RequireJobContainer = "ACTIONS_REQUIRE_JOB_CONTAINER";
```

Comment:
I have `ACTIONS_RUNNER_REQUIRE_JOB_CONTAINER` in my ADR, do you have a preference for this? Looks like we use a mix of both in the runner